_config.yml

```diff
@@ -37,3 +37,8 @@ compress_html:
   startings: []
   blanklines: false
   profile: false
+
+# Google Analytics Tracking (optional)
+# e.g, UA-1234567-89
```

Comment:
The example it's not necessary. The reference is described by the Theme Documentation.